### PR TITLE
Немченко Евгений 2538

### PR DESCRIPTION
**Pull request by eugene536**

### CODE REVIEW

File: app/src/main/java/ru/eugene/extratask1/ViewPhoto.java [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+        // as you specify a parent activity in AndroidManifest.xml.
+        int id = item.getItemId();
+
+        //noinspection SimplifiableIfStatement
+        if (id == R.id.wallpaper) {
+            WallpaperManager wallpaperManager = WallpaperManager.getInstance(this);
+            try {
+                photo.buildDrawingCache();
+                wallpaperManager.setBitmap(photo.getDrawingCache());
+                Toast.makeText(this, "Set wallpaper successfully!", Toast.LENGTH_LONG).show();
```
Comment (svasilinets) [inline app/src/main/java/ru/eugene/extratask1/ViewPhoto.java:122]: resources?

File: app/src/main/java/ru/eugene/extratask1/PhotoActivity.java [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+    }
+
+    @Override
+    public void onSaveInstanceState(Bundle savedInstanceState) {
+
+        // Save UI state changes to the savedInstanceState.
+        // This bundle will be passed to onCreate if the process is
+        // killed and restarted.
+
+        savedInstanceState.putLong("curPage", curPage);
```
Comment (svasilinets) [inline app/src/main/java/ru/eugene/extratask1/PhotoActivity.java:96]: constants?

**Comment by svasilinets** [comment]
- 1. Inaccurate code
- 3. Infinite progress in offline modes
- 1. UI: HUGE BUTTONS

20 - 15 = 5.

**Comment by eugene536** [comment]
20 - 5 = 15 :)

**Pull request closed without merging**